pages/webview/base.py

```diff
@@ -74,6 +76,31 @@ def offscreen_click(self, element):
         element.send_keys(Keys.ENTER)
         return element
 
+    def get_window_size(self, dimension=None):
+        """Return the current window dimensions."""
+        get_size = self.driver.get_window_size()
+        if not dimension:
+            return get_size
+        if dimension not in get_size:
+            raise IndexError('Unknown dimension: %s' % dimension)
+        return get_size[dimension]
+
+    def set_window_size(self, width=0, height=0):
+        """Attempt to change the browser window size."""
+        if width >= 1 and height >= 1:
+            self.driver.set_window_size(width, height)
+            sleep(1.0)
+
+    def set_window_position(self, x_=0, y_=0):
```

Comment:
Why `x_` instead of `x`?

Reply:
Honestly, I don't know. This are some old codes that @gregfitch wrote for automation and I just copied.